startupscript/aws/vm-metadata.sh

```diff
@@ -15,6 +15,9 @@ function get_metadata_value() {
     exit 1
   fi
   local tag_key=vwbapp:"$1"
+  
+  apt-get update
```

Comment:
Do the instances have neither `wget` nor `curl`?

I don't think installing `wget` should be inside the function. Was that an intentional choice? If so, the reasoning should be documented.

It would also be nice to use `--yes` instead of `-y` and to general prefer long options over short.

Reply:
i moved it to the post-startup.sh in the other PR

Reply:
will drop this one